pkg/kubectl/cmd/delete.go

```diff
@@ -251,7 +251,8 @@ func ReapResult(r *resource.Result, f cmdutil.Factory, out io.Writer, isDefaultD
 		if err != nil {
 			// If there is no reaper for this resources and the user didn't explicitly ask for stop.
 			if kubectl.IsNoSuchReaperError(err) && isDefaultDelete {
-				return deleteResource(info, out, shortOutput, mapper)
+				// No client side reaper found. Let the server do cascading deletion.
+				return cascadingDeleteResource(info, out, shortOutput, mapper)
```

Comment:
We should probably be cascading even for things that have reapers (reaper is a client implementation of cascade, so the intent is to cascade).  Do all existing reapers cascade?  If no, they should also orphan: false.